.github/dependabot.yml

```diff
@@ -20,6 +20,413 @@ updates:
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/cilium-test
+    schedule:
+      interval: daily
+    target-branch: "v1.10"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/clustermesh-apiserver
+    schedule:
+      interval: daily
+    target-branch: "v1.10"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/hubble-relay
+    schedule:
+      interval: daily
+    target-branch: "v1.10"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/operator
+    schedule:
+      interval: daily
+    target-branch: "v1.10"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/runtime
+    schedule:
+      interval: daily
+    target-branch: "v1.10"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+# # # # # # # # # # # # # # # #
+#  Docker images v1.9 branch  #
+# # # # # # # # # # # # # # # #
+
+  - package-ecosystem: docker
+    directory: ./images/builder
+    schedule:
+      interval: daily
+    target-branch: "v1.9"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/cache
+    schedule:
+      interval: daily
+    target-branch: "v1.9"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/cilium
+    schedule:
+      interval: daily
+    target-branch: "v1.9"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/cilium-docker-plugin
+    schedule:
+      interval: daily
+    target-branch: "v1.9"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/cilium-test
+    schedule:
+      interval: daily
+    target-branch: "v1.9"
+    open-pull-requests-limit: 5
+    rebase-strategy: "disabled"
+    labels:
+      - kind/enhancement
+      - release-note/misc
+
+  - package-ecosystem: docker
+    directory: ./images/clustermesh-apiserver
```

Comment:
This directory doesn't seem to exist in the `v1.9` branch. I thinkwe can drop this entry.

Reply:
oh nice spotting